### ISSUE TITLE
Non side-tag builds wait for incorrect repo

### ISSUE THREAD

**Issue by QuLogic** [juhp/fbrnch#14]
When I do `fbrnch parallel -B`, I see:
```
Successfully waited 2:26 for a new f33-build repo
Successfully waited 0:00 for R-cpp11-0.2.2-1.fc33~bootstrap to appear in the f33-build repo
R-cpp11-0.2.2-1.fc33~bootstrap job completed (3 jobs left)
Successfully waited 2:32 for a new f32-build repo
Successfully waited 0:00 for R-cpp11-0.2.2-1.fc32~bootstrap to appear in the f32-build repo
R-cpp11-0.2.2-1.fc32~bootstrap job completed (2 jobs left)
Successfully waited 5:16 for a new f34-build repo
Successfully waited 7:17 for a new f34-build repo
Successfully waited 0:00 for R-cpp11-0.2.2-1.fc34~bootstrap to appear in the f34-build repo
R-cpp11-0.2.2-1.fc34~bootstrap job completed (1 jobs left)
```
Why is it waiting for a new `f3?-build` repo? This sometimes takes ages, even though waiting for the package to appear would have been finished.

**Comment by juhp** [comment]
Hmm, this was a (possibly bad?) workaround for wait-repo warning about nvr is not being latest yet in the target tag.

Basically what it does now is first wait for the next new-repo and then immediately it checks for the build there.
Hence the `Successfully waited 0:00 for R-cpp11-0.2.2-1.fc3?~bootstrap to appear`.

> Why is it waiting for a new `f3?-build` repo? This sometimes takes ages, even though waiting for the package to appear would have been finished.

Can you say more about what you are seeing?  So far this had worked for me but it is quite possible there is some fault in the logic. Not sure if we are on the same page here.

**Comment by juhp** [comment]
Maybe I should handle the messaging in fbrnch to "normalize" the output.

**Comment by QuLogic** [comment]
Waiting for `f33-build` (for example) takes much much longer. I'm building something right now and can run `koji wait-repo f33-build --build=...` which returns immediately, while `fbrnch` is still waiting.

**Comment by QuLogic** [comment]
I left something running today; it waited on `f32-build`/`f31-build` for 2 hours and timed out. But the _builds_ were in the repo way before then.

**Comment by juhp** [comment]
Okay I have dropped the double wait-repo anyway in git - though it ought to have worked.  Thanks for raising this.

Anyway I now implemented and committed native wait-repo - planning to build it quite soon in copr. :-)

**Comment by juhp** [comment]
This should be fixed in the coming 0.6 build with native wait-repo.

**Comment by QuLogic** [comment]
Seems to be working much better now.
